src/components/organisms/ProfileModal/EditProfileForm/EditProfileForm.tsx

```diff
@@ -21,25 +22,25 @@ interface PropsType {
 const EditProfileForm: React.FunctionComponent<PropsType> = ({
   setIsEditMode,
 }) => {
-  const { user, users, profileQuestions } = useSelector((state: any) => ({
-    user: state.user,
-    users: state.firestore.data.users,
+  const { user, profile } = useUser();
+  const { profileQuestions } = useSelector((state: any) => ({
     profileQuestions: state.firestore.data.currentVenue.profile_questions,
   }));
   const onSubmit = async (data: ProfileFormData & QuestionsFormData) => {
+    if (!user) return;
     await updateUserProfile(user.uid, data);
     setIsEditMode(false);
   };
-  const defaultValues: EditProfileFormValuesType = {
-    partyName: users?.[user.uid]?.partyName || "",
-    pictureUrl: users?.[user.uid]?.pictureUrl || "/anonymous-profile-icon.jpeg",
+  const defaultValues = {
+    partyName: profile?.partyName,
+    pictureUrl: profile?.pictureUrl || "/anonymous-profile-icon.jpeg",
   };
 
   profileQuestions &&
     profileQuestions.map(
       (question: QuestionType) =>
-        (defaultValues[question.name] =
-          users?.[user.uid]?.[question.name] || "")
+        //@ts-ignore wtf is this
```

Comment:
😂

Reply:
if the user didn't answer the questions, profile[question.name] will be undefined

Reply:
but we already set the defaultValues to be either users answer or some default:

see
```
 partyName: users?.[user.uid]?.partyName || "",	    partyName: profile?.partyName,
    pictureUrl: users?.[user.uid]?.pictureUrl || "/anonymous-profile-icon.jpeg",
```

Reply:
we don't do it for questions